### PR TITLE
Pr131

### PR DESCRIPTION
**Pull request by zzzeek**
multi-file, multi rev pr

### CODE REVIEW

**Comment by sqla-tester** [comment]
New Gerrit review created for change 651fe5dba056259547790ae1f9f2721c1e102500: https://gerrit.sqlalchemy.org/#/c/sqlalchemy/testgerrit/+/1043

File: bar.py [hunk 1]
```diff
@@ -3,13 +3,23 @@
 from bar import bar
 from bar import bat
 
+name = None
+
 
 def bar_one(x, y):
     bar.do_x(x)
-    bar.do_y(y)
+    bar.do_y(y + 7)
```
Comment (zzzeek) [inline bar.py]: a comment on an old line...

File: bar.py [hunk 2]
```diff
@@ -3,13 +3,23 @@
...
 
 
 def bar_two(z, q):
+    bat.do_z(z)
```
Comment (zzzeek) [inline bar.py]: and a comment on a new line

File: bar.py [hunk 3]
```diff
@@ -3,13 +3,23 @@
...
+name = None
+
 
 def bar_one(x, y):
     bar.do_x(x)
-    bar.do_y(y)
+    bar.do_y(y + 7)
 
 
 def bar_two(z, q):
```
Comment (sqla-tester) [inline bar.py]: **mike bayer** (zzzeek) wrote:

comment on the old version

File: bar.py (hunk 1 above)
Comment (sqla-tester) [inline bar.py]: **mike bayer** (zzzeek) wrote:

and now the new version

File: bar.py [hunk 4]
```diff
@@ -3,13 +3,23 @@
 from bar import bar
 from bar import bat
 
+name = None
```
Comment (sqla-tester) [inline bar.py]: **mike bayer** (zzzeek) wrote:

needs to be False

File: bar.py [hunk 5]
```diff
@@ -3,13 +3,23 @@
...
-    bar.do_y(y)
+    bar.do_y(y + 7)
 
 
 def bar_two(z, q):
+    bat.do_z(z)
+    if name:
+        bat.do_q(q)
+    else:
+        bat.do_q(q + 7)
```
Comment (sqla-tester) [inline bar.py]: **mike bayer** (zzzeek) wrote:

im not sure this part works.

   bat.do_q(q + 9)

File: foo.py [hunk 6]
```diff
@@ -10,21 +10,22 @@ def foo_one(x, y):
     bar.do_y(y)
 
 
-def foo_two(z, q):
+def foo_two(z, q, x, y):
     """do foo two"""
     bat.do_z(z)
     bat.do_q(q)
+    bar.do_x(x)
```
Comment (sqla-tester) [inline foo.py]: **mike bayer** (zzzeek) wrote:

x + 10

**Comment by sqla-tester** [comment]
Patchset 9b8355f489fdf7884df66aa08f4b0bac9c5457d0 added to existing Gerrit review https://gerrit.sqlalchemy.org/#/c/sqlalchemy/testgerrit/+/1043

File: bar.py [hunk 7]
```diff
@@ -3,15 +3,20 @@
 from bar import bar
 from bar import bat
 
+name = False
+
 
 def bar_one(x, y):
     bar.do_x(x)
-    bar.do_y(y)
```
Comment (sqla-tester) [inline bar.py]: **mike bayer** (zzzeek) wrote:

this might have been correct

File: bar.py [hunk 8]
```diff
@@ -3,15 +3,20 @@
...
+    bar.do_y(y + 7)
 
 
 def bar_two(z, q):
     bat.do_z(z)
-    bat.do_q(q)
+    if name:
+        bat.do_q(q)
+    else:
+        bat.do_q(q + 7)
```
Comment (sqla-tester) [inline bar.py]: **mike bayer** (zzzeek) wrote:

make sure this is 12

File: foo.py [hunk 9]
```diff
@@ -3,28 +3,30 @@
 from foo import bar
 from foo import bat
 
+import io  # noqa
```
Comment (sqla-tester) [inline foo.py]: **mike bayer** (zzzeek) wrote:

not needed

File: foo.py [hunk 10]
```diff
@@ -3,28 +3,30 @@
...
     bar.do_y(y)
 
 
-def foo_two(z, q):
+def foo_two(z, x, y):
     """do foo two"""
-    bat.do_z(z)
-    bat.do_q(q)
+    bat.do_z(z * 12)
+    bar.do_x(x).y(y).z(z)
```
Comment (sqla-tester) [inline foo.py]: **mike bayer** (zzzeek) wrote:

really dont know about this

**Comment by sqla-tester** [comment]
Gerrit review https://gerrit.sqlalchemy.org/1043 has been **abandoned**.  That means that at least for the moment I need to close this pull request. Sorry it didn't work out :(

**Comment by sqla-tester** [comment]
Gerrit review https://gerrit.sqlalchemy.org/1043 has been **restored**.  That means I can reopen this pull request!  Hooray :)

**Comment by sqla-tester** [comment]
Gerrit review https://gerrit.sqlalchemy.org/1043 has been **merged**. Congratulations! :)

**Pull request closed without merging**